tasks/simplifications.json

```diff
@@ -437,70 +437,70 @@
 		"geomfield": "the_geom",
 		"factor": "8",
 		"maxmemory": "8192",
-		"skipfailures": "no"
+		"skipfailures": "yes"
```

Comment:
Why do we have to enable this?

Reply:
The TIGER geometries are the reason to have the `skipfailures` property in this JSON in the first place. `ogr2ogr` gives a lot of warnings but returns an error signal that makes the task fail.

Reply:
Yep but we have it disabled and now is enable. Why didn't we have it enabled and why is required now?

Reply:
We manually simplified the `obs_xxx` tables for TIGER (final ETL tables) and now we are automatically simplifying the original TIGER tables